src/components/keypad/trigonometry-page.js

```diff
@@ -0,0 +1,98 @@
+// @flow
+import React from "react";
+import {View} from "@khanacademy/wonder-blocks-core";
+import Color from "@khanacademy/wonder-blocks-color";
+
+import Keys from "../../data/key-configs";
+import {
+    KeypadPageContainer,
+    KeypadButton,
+    SecondaryKeypadButton,
+    KeypadActionButton,
+    PlaceHolderButton,
+} from "./keypad-page-items";
+
+import type {KeyConfig} from "../../data/key-configs";
+
+type Props = {|
+    onClickKey: (keyConfig: string) => void,
+|};
+type State = {};
+export default class TrigonometryInputPage extends React.Component<
+    Props,
+    State,
+> {
+    render() {
+        const {onClickKey} = this.props;
+        return (
+            <KeypadPageContainer>
+                {/* Row 1 */}
+                <SecondaryKeypadButton
+                    keyConfig={Keys.SIN}
+                    onClickKey={onClickKey}
+                />
+                <SecondaryKeypadButton
+                    keyConfig={Keys.COS}
+                    onClickKey={onClickKey}
+                />
+                <SecondaryKeypadButton
+                    keyConfig={Keys.TAN}
+                    onClickKey={onClickKey}
+                />
+
+                <SecondaryKeypadButton
+                    keyConfig={Keys.LEFT_PAREN}
+                    onClickKey={onClickKey}
+                    style={{
+                        gridColumn: 5,
+                    }}
+                />
+                <SecondaryKeypadButton
+                    keyConfig={Keys.RIGHT_PAREN}
+                    onClickKey={onClickKey}
+                    style={{
+                        gridColumn: 6,
+                    }}
+                />
```

Comment:
Nit - can we get a space before the row comments so it's easier to see where the breaks are?